spec/controllers/oauth/applications_controller_spec.rb

```diff
@@ -182,7 +186,7 @@ module Oauth
     end
 
     it "should not let a user create an application" do
-      controller.sign_in! user
+      controller.sign_in! user3
```

Comment:
only admins can create right? so can you reuse `user` here? Kind of nice that `user` has an oauth application and this would show that this user does not have create permissions.

Reply:
Updated